pulsar-broker/src/test/java/org/apache/pulsar/broker/service/InactiveTopicDeleteTest.java

```diff
@@ -303,10 +303,10 @@ public void testTopicLevelInActiveTopicApi() throws Exception {
         conf.setSystemTopicEnabled(true);
         conf.setTopicLevelPoliciesEnabled(true);
         super.baseSetup();
-        //wait for init
-        Thread.sleep(2000);
         final String topicName = "persistent://prop/ns-abc/testMaxInactiveDuration-" + UUID.randomUUID().toString();
         admin.topics().createPartitionedTopic(topicName, 3);
+        //wait for init
+        Thread.sleep(3000);
```

Comment:
Can we find a better solution here rather than `Thread.sleep`?

Reply:
I added an api to determine whether the cache is initialized